packages/core/src/lib/Hero/wizard.svg

```diff
@@ -1,5 +1,5 @@
 <svg width="241" height="210" viewBox="0 0 241 210" fill="none" xmlns="http://www.w3.org/2000/svg">
-<g clip-path="url(#clip0_956_11337)">
+<g clipPath="url(#clip0_956_11337)">
 <path d="M186.772 140.439C186.772 140.439 209.827 106.943 222.228 104.424C222.228 104.424 181.348 80.3099 154.418 79.2014C127.486 78.0908 121.875 78.1993 121.875 78.1993L114.439 77.811C114.439 77.811 108.826 77.7025 81.8957 78.8132C54.9637 79.9238 14.0859 104.036 14.0859 104.036C26.4862 106.552 49.5417 140.051 49.5417 140.051C49.5417 140.051 14.0859 174.674 14.0859 238.531C49.8629 284.619 101.383 258.378 121.617 257.775C134.599 257.387 190.471 292.58 229 246.602C229 187.493 186.776 140.439 186.776 140.439H186.772Z" fill="url(#paint0_linear_956_11337)"/>
```

Comment:
In svg file attribute should be clip-path instead of clipPath
camel case is used in tsx file.